__tests__/tile-item.test.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import TileItem from '../js/components/tile-item';
+
+describe('Tile Item component behaviour', () => {
+
+    let tileItemWrapper;
+    const globalAny: any = global;
+
+    const props = {
+        deleteOutcome: null,
+        flowKey: 'tile-item-test',
```

Comment:
There are some test utility functions for generating random strings and integers at ./test-utils.js which you may want to use instead of having to think of random words.